chain/tests/chain_test_helper.rs

```diff
@@ -97,11 +97,7 @@ where
 
 		chain.set_txhashset_roots(&mut b).unwrap();
 
-		let edge_bits = if n == 2 {
```

Comment:
`n` here is block height and not sure why we would need special handling for height 2.